File: get-started/quickstart/php/client/Posts/PostsRequestBuilderGetQueryParameters.php
<?php

namespace KiotaPosts\Client\Posts;

/**
 * Get posts
*/
class PostsRequestBuilderGetQueryParameters 
{
    /**
     * @var string|null $title Filter results by title
    */
    public ?string $title = null;
    
    /**
     * @var int|null $userId Filter results by user ID
    */
    public ?int $userId = null;
    
    /**
     * Instantiates a new PostsRequestBuilderGetQueryParameters and sets the default values.
     * @param string|null $title Filter results by title
     * @param int|null $userId Filter results by user ID
    */
    public function __construct(?string $title = null, ?int $userId = null) {
        $this->title = $title;
        $this->userId = $userId;
    }

}

File: get-started/quickstart/dart/client/posts/posts_request_builder_get_query_parameters.dart
// ignore_for_file: type=lint
import 'package:microsoft_kiota_abstractions/microsoft_kiota_abstractions.dart';

/// auto generated
/// Get posts
class PostsRequestBuilderGetQueryParameters implements AbstractQueryParameters {
    ///  Filter results by title
    String? title;
    ///  Filter results by user ID
    int? userId;
    /// Extracts the query parameters into a map for the URI template parsing.
    @override
    Map<String, dynamic> toMap() {
        return {
            'title' : title,
            'userId' : userId,
        };
    }
}
